pkg/apis/experiment/experiment.go

```diff
@@ -0,0 +1,390 @@
+		}
+
+		// Errors present
+		if len(savedExperiment.Errors) > 0 {
+			return RunExperimentData{}, errors.New(savedExperiment.Errors[0].Message)
+		}
+
+		if strings.Contains(savedExperiment.Data.Message, "experiment saved successfully") {
+			fmt.Print("\n🚀 Chaos Experiment successfully Saved 🎉")
+		}
+	} else {
+		return RunExperimentData{}, errors.New("graphql schema error")
+	}
+
+	// Query to Run the Chaos Experiment
+	runQuery := `{"query":"mutation{ \n runChaosExperiment(experimentID:  \"` + requestData.ID + `\", projectID:  \"` + pid + `\"){\n notifyID \n}}"}`
+	resp, err = apis.SendRequest(apis.SendRequestParams{Endpoint: cred.Endpoint + utils.GQLAPIPath, Token: cred.Token}, []byte(runQuery), string(types.Post))
+
+	if err != nil {
+		return RunExperimentData{}, errors.New("Error in Running Chaos Experiment: " + err.Error())
+	}
+
+	bodyBytes, err = ioutil.ReadAll(resp.Body)
+	defer resp.Body.Close()
+	if err != nil {
+		return RunExperimentData{}, errors.New("Error in Running Chaos Experiment: " + err.Error())
+	}
+
+	if resp.StatusCode == http.StatusOK {
+		var runExperiment RunExperimentData
+		err = json.Unmarshal(bodyBytes, &runExperiment)
+		if err != nil {
+			return RunExperimentData{}, errors.New("Error in Running Chaos Experiment: " + err.Error())
+		}
+
+		if len(runExperiment.Errors) > 0 {
+			return RunExperimentData{}, errors.New(runExperiment.Errors[0].Message)
+		}
+		return runExperiment, nil
+	} else {
+		return RunExperimentData{}, err
+	}
+}
+
+func SaveExperiment(pid string, requestData model.SaveChaosExperimentRequest, cred types.Credentials) (SaveExperimentData, error) {
+
+	// Query to Save the Experiment
+	var gqlReq SaveChaosExperimentGraphQLRequest
+
+	gqlReq.Query = SaveExperimentQuery
+	gqlReq.Variables.ProjectID = pid
+	gqlReq.Variables.SaveChaosExperimentRequest = requestData
+
+	query, err := json.Marshal(gqlReq)
+	if err != nil {
+		return SaveExperimentData{}, err
+	}
+
+	resp, err := apis.SendRequest(
+		apis.SendRequestParams{
+			Endpoint: cred.Endpoint + utils.GQLAPIPath,
+			Token:    cred.Token,
+		},
+		query,
+		string(types.Post),
+	)
+	if err != nil {
+		return SaveExperimentData{}, err
+	}
+
+	bodyBytes, err := ioutil.ReadAll(resp.Body)
+
+	defer resp.Body.Close()
+	if err != nil {
+		return SaveExperimentData{}, errors.New("Error in creating Chaos Experiment: " + err.Error())
+	}
+
+	if resp.StatusCode == http.StatusOK {
+		var savedExperiment SaveExperimentData
+
+		err = json.Unmarshal(bodyBytes, &savedExperiment)
+
+		if err != nil {
+			return SaveExperimentData{}, errors.New("Error in creating Chaos Experiment: " + err.Error())
+		}
+
+		// Errors present
+		if len(savedExperiment.Errors) > 0 {
+			return SaveExperimentData{}, errors.New(savedExperiment.Errors[0].Message)
+		}
+
+		if strings.Contains(savedExperiment.Data.Message, "experiment saved successfully") {
+			fmt.Print("\n🚀 Chaos Experiment successfully Saved 🎉")
```

Comment:
Saved -> saved
please make this change everywhere

Reply:
I've excluded this statement as cmd will handle it now.